EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+---
+eip: 2502
+title: µTopic Delegation
+author: Ricardo Guilherme Schmidt (@3esmit)
+discussions-to: https://ethereum-magicians.org/t/eip-2502-topic-delegation/3974
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-02
+requires: 2470
+
+---
+
+## Simple Summary
+
+A common trust network for ethereum.
+
+A general purpose hierarchy of delegations which can divided into micro topics. 
+
+## Abstract
+
+When "everything" becomes DAOable, it could become a problem to keep track of all the micro DAOs. Just like ERC20 have an allowance for other address spent on their behalf, 
+Cold wallets usually don't want to be unlocked very often to be able to vote. 
+Users might want to trust all their DAOs to other address for vote for them. 
+A group of people want to trust each other to vote in topics.
+DAOs might want to inherit the _root_ delegate of users for their own _topic_, where users could define a specific delegate..
+
+## Motivation
+
+Convergence of user delegations in a common trust network, allowing to specify specific delegates for specific sub topics.
+
+Allows proposals to inherit a common and specialized trust network.
+
+## Specification
+
+This EIP defines the delegation behavior, a common interface for the delegations, the relationship between delegations hierarchy, and the address/deploy of the root delegation.
+
+Every address can delegate to another address, by default, at root delegation, all users delegate to themselves:
+
+```plantuml
+@startuml
+:A: -> :A:
+@enduml
+```
+
+When a user delegate to itself, it represents a circular delegation of size 0, the delegations always end up in a circular delegations, which can have other size, as of size 1:
+
+```plantuml
+@startuml
+:A: -> :B:
+:A: <- :B:
+@enduml
+```
+
+A circular delegation of size 2 would be a triangular delegation:
+
+```plantuml
+@startuml
+:A: -> :B:
+:B: -> :C:
+:C: -> :A:
+@enduml
+```
+
+Ultimately, all delegations paths will end in a circular delegation, as example, a delegation chain ending in a circular delegation of size 0:
+
+
+```plantuml
+@startuml
+:A: -> :B:
+:B: -> :B:
+@enduml
+```
+
+Delegations chains may also end in a circular delegation of other sizes:
+
+```plantuml
+@startuml
+:A: -> :B:
+:B: -> :C:
+:C: -> :D:
+:D: -> :E:
+:E: -> :F:
+:F: -> :G:
+:G: -> :D:
+:H: --> :G:
+:I: --> :B:
+@enduml
+```
+
+### Influence flow
+
+The delegate which gets the influence is the first on the chain that vote, in the example above, if only `E` voted, it would accumulate the influence of all others and itself, `A,B,C,D,E,F,G,I`, while if `D` and `G` voted, then  `D` would be able to claim influence of `A,B,I,C,D`, while `G` would claim from `E,F,H,G`. 
```

Comment:
```suggestion
The delegate which gets the influence is the first on the chain that vote, in the example above, if only `E` voted, it would accumulate the influence of all others and itself, `A,B,C,D,E,F,G,H,I`, while if `D` and `G` voted, then  `D` would be able to claim influence of `A,B,I,C,D`, while `G` would claim from `E,F,H,G`.
```